src/apps/common/submodules/numbers/numbers.js

```diff
@@ -1936,6 +1936,15 @@ define(function(require) {
 					phoneNumber: phoneNumber
 				},
 				success: function(_data, status) {
+					if (_.get(_data, 'metadata.carrier_module')) {
+						_data.data.carrier_module = _.get(_data, 'metadata.carrier_module');
+					}
+					if (_.get(_data, 'metadata.features')) {
+						_data.data['metadata'] = {
+							features: _.get(_data,'metadata.features')
+						};
+					}
```

Comment:
I would suggest to add all the metadata into the metadata property:
```suggestion
						_data.data['metadata'] = _.get(_data, 'metadata', {})
```